### PR TITLE
Removing FabricJS references from website

### PR DESCRIPTION
**Pull request by lynamemi**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [X] Include a change request file using `$ npm run change`

#### Description of changes

Removed all references of FabricJS across the website.

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by Jahnp** [comment]
Thanks for doing this. The code looks good, but given the breadth of changes, I'll try to do a once-over on your branch locally before merging this in.

File: packages/office-ui-fabric-react/src/components/Callout/CalloutPage.tsx [hunk 1]
```diff
@@ -60,9 +60,6 @@ export class CalloutPage extends React.Component<IComponentDemoPageProps, any> {
             <p>Real-world examples of this implementation can be seen in administrative interfaces where a particularly difficult-to-understand concept is paired with the iconClassNames.info "i" icon. In this example, Callout - with its tip text - is opened when the user clicks on or hovers over the icon.</p>
           </div>
         }
-        related={
```
Comment (Jahnp) [inline packages/office-ui-fabric-react/src/components/Callout/CalloutPage.tsx]: Did this for sure remove the "also available in" links? Those still seem to show up locally for me when I tested against this branch.

Comment (Jahnp) [inline packages/office-ui-fabric-react/src/components/Callout/CalloutPage.tsx]: Ah--likely false alarm. I didn't rebuild `office-ui-fabric-react` after switching to this branch; trying that now.

Comment (Jahnp) [inline packages/office-ui-fabric-react/src/components/Callout/CalloutPage.tsx]: Yup, false alarm. Rebuilt and confirmed these are gone now.

**Comment by johannao76** [comment]
For us newbies, can you provide more context on what this change is and why it is needed?

**Comment by Jahnp** [comment]
@johannao76, I'll send you mail offline.

File: packages/office-ui-fabric-react/src/components/Callout/CalloutPage.tsx (hunk 1 above)
Comment (lynamemi) [inline packages/office-ui-fabric-react/src/components/Callout/CalloutPage.tsx]: I was still having trouble with my cache / build in Edge but got them to show up completely removed in Chrome.  Glad to hear a confirmation it worked for you, too!

**Comment by lynamemi** [comment]
@Jahnp just talked to @battletoilet about our new website v 5.0 branch. Given the timing, it looks like I'll have to shift this pull request over to that branch. Hopefully won't get to whacky.

**Comment by micahgodbolt** [comment]
image

**Pull request merged**